azurelinuxagent/common/cgroup.py

```diff
@@ -241,11 +243,16 @@ def get_throttled_time(self):
         return float(self._current_throttled_time - self._previous_throttled_time) / 1E9
 
     def get_tracked_metrics(self, **kwargs):
-        tracked = [
-            MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.PROCESSOR_PERCENT_TIME, self.name, self.get_cpu_usage()),
-        ]
+        tracked = []
+        cpu_usage = self.get_cpu_usage()
+        if cpu_usage >= float(0):
+            tracked.append(MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.PROCESSOR_PERCENT_TIME, self.name, cpu_usage))
+
         if 'track_throttled_time' in kwargs and kwargs['track_throttled_time']:
-            tracked.append(MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.THROTTLED_TIME, self.name, self.get_throttled_time()))
+            throttled_time = self.get_throttled_time()
+            if cpu_usage >= float(0) and throttled_time >= float(0):
```

Comment:
In this case, very likely throttled time could have wrong value too. So, I decided not to report both.

Reply:
I think we do not need to check for the 2 conditions, if the single condition cpu_usage < 0 is true, we should not report the 2 metrics

Reply:
yeah, if cpu_usage < 0 is true then it won't check second condition. I added second condition for rare case.